modules/installation-creating-azure-control-plane.adoc

```diff
@@ -36,11 +36,6 @@ If you do not use the provided ARM template to create your control plane machine
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
-* Create and configure networking and load balancers in {cp}.
-* Create control plane and compute roles.
 * Create the bootstrap machine.
```

Comment:
Same comment as above

Reply:
This is the first instance of 'Create the bootstrap machine' in the assembly, so I left it here and removed it from later modules, (e.g., [Wait for bootstrap completion and remove bootstrap resources](https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-azure-user-infra-wait-for-bootstrap_installing-azure-user-infra) module).